File: Models/Draft.cs
namespace ProjectOlympia;

public class Draft
{
    public required Guid Id { get; set; }
    public required string Name { get; set; }
    public required List<User> Users { get; set; }
    public required List<Athlete> Athletes { get; set; }
    public required EDraftStatus Status { get; set; }
    public required string DraftOrder { get; set;}

    public Draft()
    {}
}

File: ClientApp/src/app/models/draft.ts
import { User } from "./user";
import { Athlete } from "./athlete";
import { EDraftStatus } from "./e-draft-status";

export interface Draft {
    id: string;
    name: string;
    users: User[];
    athletes: Athlete[];
    status: EDraftStatus;
    draftOrder: string[];
}